### PR TITLE
Docs: Clarify data streaming with plugins documentation

### PR DESCRIPTION
**Pull request by FZambia**
Comma here can be a bit frustrating.

### CODE REVIEW

**Review by osg-grafana — CHANGES_REQUESTED** [review]
Please revert this change.

**Comment by osg-grafana** [comment]
Please say more about the confusion that this is causing. We can certainly reword it, but the change introduces a bug.

File: docs/sources/live/live-feature-overview.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ As soon as there is a change to the dashboard layout, it is automatically reflec
 
 ## Data streaming from plugins
 
-With Grafana Live data source, plugins can stream data updates in the form of Grafana data frames to a frontend.
+With a help of Grafana Live data source plugins can stream data updates in the form of Grafana data frames to a frontend.
```
Comment (ryantxu) [inline docs/sources/live/live-feature-overview.md:18]:
```suggestion
With Grafana Live, backend data source plugins can stream updates to frontend panels.
```

maybe?

Comment (osg-grafana) [reply docs/sources/live/live-feature-overview.md:18]: @FZambia, does this suggestion clarify things?

Comment (FZambia) [reply docs/sources/live/live-feature-overview.md:18]: Yes, previously reader could think that there is some sort of `Grafana Live data source` which is confusing.

**Comment by FZambia** [comment]
@osg-grafana hello, applied Ryan's suggestion.

**Review by osg-grafana — APPROVED** [review]

**Pull request merged**